src/utils.ts

```diff
@@ -1,8 +1,17 @@
+import crypto from 'crypto';
```

Comment:
You don't need a cryptographic rng for that. This is a big dependency for a front-end project. Do you see any issue with an auto-increment id?

Reply:
How will you know the current value?

Reply:
```js
let currentValue = 1;

function getNextId() {
  return ++currentValue;
}
```

Reply:
I send it through the context and keep it in the state, shouldn't be lost in your proposal?

Reply:
You misunderstand me, I think. I'm just suggesting to change `export function randomString() [}` with `export function getNextId() {}`, nothing else.